user/languages/javascript-with-nodejs.md

```diff
@@ -245,60 +293,18 @@ Find the source code at [travis-ci-meteor-packages](https://github.com/arunoda/t
 For JavaScript/Node.js projects, `env` and `node_js` can be used as arrays
 to construct a build matrix.
 
-## Even more Nodejs versions
-
-If you need more specific control of Node.js version in your build, use any of
-the following available versions. Releases not shown in this list may be used if
-`nvm` can install them.
-
-- 7.7.x
-- 6.1.x
-- 6.0.x
-- 5.11.x
-- 5.10.x
-- 5.9.x
-- 5.8.x
-- 5.7.x
-- 5.6.x
-- 5.5.x
-- 5.4.x
-- 5.3.x
-- 5.2.x
-- 5.1.x
-- 5.0.x
-- 4.4.x
-- 4.3.x
-- 4.2.x
-- 4.1.x
-- 4.0.x
-- 0.12.x
-- 0.11.x
-- 0.10.x
-- 0.8.x
-- 0.6.x
-{: .column-3}
-
-Specifying only a major (e.g., "7") or major.minor version (e.g., "7.7") will run using the
-latest published patch release for that version such as "7.7.1".
-[nvm](https://github.com/creationix/nvm) handles version resolution, so any
-version or [alias](https://github.com/creationix/nvm#usage) of Node.js or io.js
-that nvm can install is available.
-
-If your `.travis.yml` specifies a version of Node.js that `nvm` cannot install,
-the job errors immediately. For example, see [hook.io-amqp-listener
-.travis.yml](https://github.com/scottyapp/hook.io-amqp-listener/blob/master/.travis.yml).
-
-For precise versions pre-installed on the VM, please consult "Build system
-information" in the build log.
-
 ## Node.js v4 (or io.js v3) compiler requirements
 
-To compile native modules for io.js v3 or Node.js v4 or later, a
-[C++11 standard](https://en.wikipedia.org/wiki/C%2B%2B11)-compliant compiler is required. More specifically, either gcc 4.8 (or later), or clang 3.5 (or later) works.
+To compile native modules for io.js v3 or Node.js v4 or older, a [C++11
```

Comment:
It was, but perhaps I was mistaken. It followed on from the old Node versions topic. But this is actually compiling modules not older versions.